src/palace/manager/service/google_drive/configuration.py

```diff
@@ -7,8 +7,13 @@
     ServiceConfiguration,
 )
 
-# Environment variable specifying the google drive root folder for this environment
-PALACE_GOOGLE_DRIVE_ROOT_ENVIRONMENT_VARIABLE = "PALACE_GOOGLE_DRIVE_ROOT"
+# Environment variable specifying the google drive parent folder ID for this environment.
+# This value may be a folder ID or a shared drive ID. For example, the google drive folder URI will look like
+# https://drive.google.com/drive/u/1/folders/0AGtlKYStJaC3Uk9PVZ .   "0AGtlKYStJaC3Uk9PVZ" (not a real
+# folder ID) is the value that should be assigned environment variable.
+PALACE_GOOGLE_DRIVE_PARENT_FOLDER_ID_ENVIRONMENT_VARIABLE = (
+    "PALACE_GOOGLE_DRIVE_PARENT_FOLDER_ID"
```

Comment:
Need to update the `README` with the new environment variable, and the information you put here in a comment.